css/civicrm.css

```diff
@@ -125,17 +125,39 @@ input.crm-form-checkbox + label {
   vertical-align: top;
 }
 /* Styles for Sticky Header */
+.crm-sticky thead > tr > th,
 .crm-container table thead.sticky {
   background-color: #fff;
 }
 
 .crm-container thead div.sticky-header {
   height: 15px;
+}
+
+.crm-sticky thead,
+.crm-container thead div.sticky-header {
   border-bottom: 2px solid #cfcec3;
   background-color: #fff;
   z-index: 10;
 }
 
+.crm-sticky thead {
+  position: sticky;
+  top: var(--crm-menubar-bottom);
```

Comment:
Should this be `--crm-sticky-top`? Otherwise I can't see this sit anywhere - tho then it would only be set in WordPress.

Reply:
I haven't checked the other code actually. I was thinking about creating a master ticket (just haven't had time) about reworking all the tables. I have a few ideas of some things we can do to streamline things a bit. I'll grep (in about a week) for the class you mentioned and happy to change what I wrote to something already defined. I just didn't find anything before I think.

Reply:
No I think I misunderstood your comment above and thought you were indicating that in fact it was a duplication. Wordpress CiviCRM generates this automatically based on the resizing of the window. The same variable exists for D9 (probably onward too). My thinking was to use the bottom, because with a sticky header it should essentially start at that bottom (below the actual menu) when the user gets to the point where it should be sticky.

Reply:
Ah I see! Had missed that Coleman [added that](https://github.com/civicrm/civicrm-core/blob/973f77a69233a23150cc5562ee1c19f68fe326cd/js/crm.menubar.js#L527) last year as a JS generated variable.